src/fieri/app/models/cookbook_artifact.rb

```diff
@@ -113,7 +113,7 @@ def binary?(filepath)
       # need to check for application/xml explicitly because only versions of
       # libmagic < 5.26 are available on Ubuntu 14.04 and 16.04 which are our
       # CI (Travis and Automate) node platforms
-      !(magic.file(filepath) =~ %r{^(text\/|application\/xml)})
+      !(magic.file(filepath) =~ %r{^(text\/|inode\/x-empty|application\/xml)})
```

Comment:
I think I'll want to refactor this bit the next time something gets added to the regex to declare yet another thing "not binary."